Modify Optee-OS RAM locations in conf.mk to work with 256MB RAM.

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 0b6f454..e59fa6b 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -279,11 +279,11 @@ endif


 ifneq ($(filter $(CFG_EMBED_DTB_SOURCE_FILE),$(flavorlist-512M)),)
-CFG_DRAM_SIZE    ?= 0x20000000
+CFG_DRAM_SIZE    ?= 0x10000000
 endif

 CFG_DRAM_BASE    ?= 0xc0000000
-CFG_DRAM_SIZE    ?= 0x40000000
+CFG_DRAM_SIZE    ?= 0x10000000
 ifeq ($(CFG_STM32MP15)-$(CFG_WITH_PAGER),y-y)
 CFG_TZSRAM_START ?= 0x2ffc0000
 CFG_TZSRAM_SIZE  ?= 0x0003f000
